core/src/main/kotlin/org/jetbrains/kotlinx/dataframe/api/cumSum.kt

```diff
@@ -79,25 +112,46 @@ private val supportedClasses = setOf(
 
 // region DataFrame
 
+/**
+ * {@include [CumSumDocs]}
+ * {@set DATA_TYPE [DataFrame]}.
```

Comment:
Unclear what "the values in the DataFrame" means. It's run on each column individually, and we get a new df with the new cumulative sum columns